src/bridge/mod.rs

```diff
@@ -236,25 +252,52 @@ async fn launch(handler: NeovimHandler, grid_size: Option<Dimensions>) -> Result
     res.map(|()| session)
 }
 
-async fn run(session: NeovimSession) {
-    match session.io_handle.await {
-        Err(join_error) => error!("Error joining IO loop: '{}'", join_error),
-        Ok(Err(error)) => {
-            if !error.is_channel_closed() {
-                error!("Error: '{}'", error);
```

Comment:
I will add this message back. I thought it wasn't important, but it's actually, the message printed in this bug for example: 
* https://github.com/neovide/neovide/issues/2452